.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: Stale PR tracking
+
+on:
+  schedule:
+    - cron: '0 9 * * 1-5'
```

Comment:
For mere mortals like me who can't read cron schedules.

https://crontab.guru/#0_9_*_*_1-5

> “At 09:00 on every day-of-week from Monday through Friday.”